rest/blip_api_test.go

```diff
@@ -1487,40 +1488,83 @@ func TestBlipDeltaSyncPull(t *testing.T) {
 
 	defer base.SetUpTestLogging(base.LevelTrace, base.KeyAll)()
 
-	bt, err := NewBlipTesterFromSpec(BlipTesterSpec{
-		restTester: &RestTester{
-			DeltaSyncEnabled: base.IsEnterpriseEdition(),
-		},
-	})
-	assert.NoError(t, err, "Error creating BlipTester")
-
-	client, err := NewBlipTesterClient(bt)
+	rt := RestTester{DeltaSyncEnabled: base.IsEnterpriseEdition()}
+	client, err := NewBlipTesterClient(&rt)
 	assert.NoError(t, err)
 	defer client.Close()
 
-	client.Deltas = true
-	client.Start()
+	client.ClientDeltas = base.IsEnterpriseEdition()
```

Comment:
Should leave this as client.ClientDeltas=true, to cover both cases (i.e. when test is run in community vs enterprise).